test/uk/gov/hmrc/nisp/connectors/IdentityVerificationConnectorSpec.scala

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2016 HM Revenue & Customs
+ * Copyright 2019 HM Revenue & Customs
```

Comment:
Did you know that you can review changes with `git diff` before adding and commiting? It can help prevent mistakes like this.